redis/scan.go

```diff
@@ -496,9 +500,13 @@ var (
 	errScanSliceValue = errors.New("redigo.ScanSlice: dest must be non-nil pointer to a struct")
 )
 
-// ScanSlice scans src to the slice pointed to by dest. The elements the dest
-// slice must be integer, float, boolean, string, struct or pointer to struct
-// values.
+// ScanSlice scans src to the slice pointed to by dest.
+//
+// If the target is a slice of types which implement Scanner then the custom
+// RedisScan method is used otherwise the following rules apply:
+//
+// The elements the dest slice must be integer, float, boolean, string, struct
```

Comment:
```suggestion
// The elements in the dest slice must be integer, float, boolean, string, struct
```

Reply:
Thanks @danmrichards fixed by https://github.com/gomodule/redigo/pull/491 as this was already merged.